05LessonFive/LessonFive.js

```diff
@@ -1,20 +1,43 @@
-module.exports = function() {
+this.loopIt = function () {
```

Comment:
LessonFive tests are not running due to a syntax error.

```
$ mocha
LessonFive.js:45
});
^

SyntaxError: Unexpected token }
```

Compare the structure of this file to the structure in other test files (such as Lesson04.js).